WordPress/src/main/res/layout/reader_interest_item_new.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<com.google.android.material.chip.Chip xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/chip"
+    style="@style/ReaderExpandableTagsViewChipNew"
```

Comment:
💡 Suggestion: If this chip is changing the `textColor`, `textSize`, and `chipBackgroundColor`, would it make sense to create a new style for it derived from the `ReaderExpandableTagsViewChipNew` style?

Reply:
Makes sense. Done 👍